source/includes/indexes/multikey.rb

```diff
@@ -0,0 +1,37 @@
+require 'mongo'
+
+# Replace the placeholders with your credentials
+uri = "<connection string>"
+
+# Sets the server_api field of the options object to Stable API version 1
+options = { server_api: { version: "1" }}
+
+# Creates a new client and connect to the server
+client = Mongo::Client.new(uri, options)
+
+# start-sample-data
+database = client.use('sample_mflix')
+collection = database[:movies]
+# end-sample-data
+
+# start-index-multikey
+# Creates an index on the "cast" field
+collection.indexes.create_one({ cast: 1 })
+# end-index-multikey
```

Comment:
I'd move the descriptive comment outside of what gets included – we introduce the examples in the text so I think this is redundant.

```suggestion
# Creates an index on the "cast" field
# start-index-multikey
collection.indexes.create_one({ cast: 1 })
# end-index-multikey
```